src/c_api/ndarray_c_api.cc

```diff
@@ -52,6 +52,36 @@ MATXSCRIPT_REGISTER_GLOBAL("runtime.NDArrayContiguous").set_body([](PyArgs args)
   return data.Contiguous();
 });
 
+MATXSCRIPT_REGISTER_GLOBAL("runtime.NDArrayReshape").set_body([](PyArgs args) -> RTValue {
+  NDArray data = args[0].As<NDArray>();
+  std::vector<int64_t> shape;
+  switch (args[1].type_code()) {
+    case TypeIndex::kRuntimeList: {
+       auto it = args[1].AsObjectRefNoCheck<List>();
+       for(auto &e:it){
+          shape.push_back(e.As<int64_t>());
+       }
+    }break;
+    case TypeIndex::kRuntimeFTList: {
+       auto it = args[1].AsObjectRefNoCheck<FTObjectBase>();
```

Comment:
Maybe just need to support FTList[int]: As<FTList<int64>>()

Reply:
fixed